### ISSUE TITLE
Numpy type is inferred as NoseTester

### ISSUE THREAD

**Issue by OopsYao** [microsoft/pyright#1430]
**Describe the bug**
Type `numpy.ndarray` is inferred as `NoseTester`.

**To Reproduce**
Install coc-pyright via coc.nvim in nvim.

**Expected behavior**
It should not report this type issue.

**Screenshots or Code**
Code in screenshot
```python
import numpy as np

a = np.pi / np.pi
```
![2021-01-28-115000_774x350_scrot](https://user-images.githubusercontent.com/32776639/106088263-9c2ece80-6160-11eb-8a07-b85b19d558cb.png)

**VS Code extension or command-line**
Vim extension coc-pyright v1.1.105 (not using ALE)

**Additional context**
Some versions:
Python version 3.9.1
numpy version 1.19.5
neovim version 0.4.4
coc.nvim version 0.0.80
OS: arch linux

About the config, I'm using the default one (no additional configuration).

**Comment by tamuhey** [comment]
duplicate: #1417

**Comment by erictraut** [comment]
Yes, this is a duplicate. Pyright is doing the right thing here. The latest version of numpy reportedly addresses this issue.
